test/db/anal/x86_64

```diff
@@ -3208,19 +3208,19 @@ c 0x5f8 case.0x59d.9
 c 0x606 case.0x59d.11
 c 0x614 case.0x59d.12
 c 0x622 case.0x59d.0
-0x00000580 0x0000058d 01:0090 13 j 0x00000622 f 0x0000058d
-0x0000058d 0x0000059f 01:0092 18 s 0x00000622 s 0x000005b2 s 0x000005c0 s 0x000005ce s 0x000005dc s 0x000005ea s 0x000005f8 s 0x00000606 s 0x00000614 s 0x0000059f
-0x0000059f 0x000005ab 01:0096 12 j 0x000005ab
-0x000005ab 0x000005b2 01:0097 7
-0x000005b2 0x000005c0 01:0099 14 j 0x000005ab
-0x000005c0 0x000005ce 01:009A 14 j 0x000005ab
-0x000005ce 0x000005dc 01:009B 14 j 0x000005ab
-0x000005dc 0x000005ea 01:009C 14 j 0x000005ab
-0x000005ea 0x000005f8 01:009D 14 j 0x000005ab
-0x000005f8 0x00000606 01:009E 14 j 0x000005ab
-0x00000606 0x00000614 01:009F 14 j 0x000005ab
-0x00000614 0x00000622 01:00A0 14 j 0x000005ab
-0x00000622 0x00000633 01:00A1 17 j 0x000005ab
+0x00000580 0x0000058d 00:0000 13 j 0x00000622 f 0x0000058d
```

Comment:
looks wrong

Reply:
This is 489215f, I think this should be the correct behavior

Reply:
Do we have any test that is actually testing that the tracing is not broken? im mainly worried about the change in behaviour here. its fine to have zero es around if this tracing info is generated by an automatic analysis command. if theres no such test. can you add it? 

Thanks!!